TestFoundation/TestHTTPCookieStorage.swift

```diff
@@ -246,6 +252,23 @@ class TestHTTPCookieStorage: XCTestCase {
         XCTAssertEqual(storage.cookies(for: url1!)!.count, 0)
     }
 
+    func descriptionCookie(with storageType: _StorageType) {
```

Comment:
As this function is only used inside `test_descriptionCookie()` why not make it a nested function?

Reply:
@ianpartridge I see that this pattern (calling a function XXX two times from test_XXX) is used in all the tests in TestHTTPCookieStorage. Do you think doing this as a separate PR will make more sense?